pandas/tests/plotting/test_frame.py

```diff
@@ -204,6 +204,17 @@ def test_color_and_style_arguments(self):
         # if there is a color symbol in the style strings:
         with pytest.raises(ValueError):
             df.plot(color=["red", "black"], style=["k-", "r--"])
+            # k for black, r for red
+
+    @pytest.mark.parametrize("color", ["green", ["yellow", "red", "green", "blue"]])
+    def test_color_and_marker(self, color):
+        # GH 21003
+        df = DataFrame(np.random.random((7, 4)))
+        ax = df.plot(color=color, style="d--")  # d for diamond
+        green_line = ax.lines[2]
+        assert green_line.get_color() == "green"
+        assert green_line.get_marker() == "d"
+        assert green_line.get_linestyle() == "--"
```

Comment:
since you might need another rebase, just one minor comment, as mentioned in https://github.com/pandas-dev/pandas/pull/33821#discussion_r426685765

I still prefer to not test only green line but all lines using parametrization, i had an example there

Reply:
> since you might need another rebase, just one minor comment, as mentioned in [#33821 (comment)](https://github.com/pandas-dev/pandas/pull/33821#discussion_r426685765)

Why do I need another rebase? I just merged upstream/master 6 days ago which unfortunately left one CI issue. I don't think another rebase will solve that?

> I still prefer to not test only green line but all lines using parametrization, i had an example there

Well the function is parametrized I'm testing the inputs `color="green"` and `color=["yellow", "red", "green", "blue"]` . The only thing is, to simplify the code, I'm only checking the third line, the one that is green for both inputs in the parametrize. I'm confident that this is enough. Checking all four lines would be overkill. After all, checking all four lines (as opposed to the third) would only guard against absurd bugs such as if we suddenly had code that ignored the `color` kwarg and made every line green.

Reply:
> I just merged upstream/master 6 days ago which unfortunately left one CI issue. I don't think another rebase will solve that?

Might do if upstream/master has changed

Regarding the test - unfortunately, absurd bugs do sometimes happen. FWIW, I think the suggestion to check all lines' colours is sensible